broker/pom.xml

```diff
@@ -171,6 +176,14 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-scheduler</artifactId>
+      <version>${project.version}</version>
+      <type>test-jar</type>
```

Comment:
:question: Why do we need the test-jar?

Reply:
For stuff in `io.camunda.zeebe.scheduler.testing`, for example the `ActorSchedulerRule` which is only useful for tests so I didn't want to include them in the regular jar (= in the `main` source set)